docs/contributing/7_Run_the_Website.md

```diff
@@ -4,29 +4,39 @@ This guide will help you set up and run the OWASP MAS website locally on your ma
 
 ## Using Docker
 
-The easiest way to run the website is by using Docker:
+The easiest way to run the website is by using Docker. The following commands will clone the necessary repositories, build the Docker image, and run the website, which will be accessible at [http://localhost:8000](http://localhost:8000).
 
 ```bash
 git clone https://github.com/OWASP/mastg.git
-cd mastg
-docker build . -t mastg
-docker run --name mastg -it --rm -p 8000:8000 -u $(id -u):$(id -g) -v $(pwd):/workspaces/mastg mastg
+git clone https://github.com/OWASP/masvs.git
+git clone https://github.com/OWASP/maswe.git
+git clone https://github.com/OWASP/mas-website.git
+cd mas-website
+docker build . -t mas-website
+docker run --rm -it \
+  -p 8000:8000 \
+  -u $(id -u):$(id -g) \
+  -e GITHUB_TOKEN \
+  -v "$PWD":/workspaces/mas-website \
+  -v "$REPO_ROOT/mastg":/workspaces/mastg \
+  -v "$REPO_ROOT/masvs":/workspaces/masvs \
+  -v "$REPO_ROOT/maswe":/workspaces/maswe \
```

Comment:
For the given instructions, this needs to change to the following to "insta-work":
```suggestion
  -v "../mastg":/workspaces/mastg \
  -v "../masvs":/workspaces/masvs \
  -v "../maswe":/workspaces/maswe \
```